pytorch_blade/pytorch_blade/compiler/mlir/converters/torch_mlir_op_filter.cpp

```diff
@@ -167,21 +168,39 @@ const std::unordered_set<std::string> &GetTorchMlirWhiteList() {
       "torch_blade::fake_quant"
     };
 
+  auto readListFromEnvString = [](const std::string& env_name) -> std::unordered_set<std::string> {
+    auto custom_ops = env::ReadStringFromEnvVar(env_name.c_str(), "");
+    std::istringstream f(custom_ops);
+    std::string s;
+    std::unordered_set<std::string> ret;
+    for (auto s : StrSplit(custom_ops, ';')) {
+      ret.insert(std::string(s));
```

Comment:
No need to make a copy of `s`? Since `StrSplit` returns `std::vector<std::string>`.